AdobeIms/Model/Config.php

```diff
@@ -18,13 +17,13 @@
  */
 class Config implements ConfigInterface
 {
-    private const XML_PATH_API_KEY = 'adobe_ims/integration/api_key';
-    private const XML_PATH_PRIVATE_KEY = 'adobe_ims/integration/private_key';
-    private const XML_PATH_TOKEN_URL = 'adobe_ims/integration/token_url';
-    private const XML_PATH_AUTH_URL_PATTERN = 'adobe_ims/integration/auth_url_pattern';
-    private const XML_PATH_LOGOUT_URL_PATTERN = 'adobe_ims/integration/logout_url';
-    private const XML_PATH_DEFAULT_PROFILE_IMAGE = 'adobe_ims/integration/default_profile_image';
-    private const XML_PATH_IMAGE_URL_PATTERN = 'adobe_ims/integration/image_url';
```

Comment:
Is this necessary ? to change const ?

Reply:
I made it public so I can access the same constants from the associated unit test class instead of duplicating those strings.

Reply:
Please do not change constants access level only for testing purposes, create required private constants in tests instead